src/pynwb/ndx_photometry/fibers_table.py

```diff
@@ -6,52 +6,55 @@
 from pynwb.core import VectorIndex
 
 
-@docval({'name': 'excitation_source',
-         'type': int,
-         'doc': 'references rows of ExcitationSourcesTable',
-         'default': None},
-        {'name': 'photodetector',
-         'type': int,
-         'doc': 'references rows of PhotodetectorsTable',
-         'default': None},
-        {'name': 'fluorophores',
-         'doc': 'references rows of FluorophoresTable',
-         'type': 'array_data',
-         'default': None,
-         'shape':(None,)},
-        {'name': 'location',
-         'type': str,
-         'doc': 'location of fiber',
-         'default': None},
-        {'name': 'notes',
-         'type': str,
-         'doc': 'description of fiber',
-         'default': None},
-        {'name': 'fiber_model_number',
-         'type': str,
-         'doc': 'fiber model number',
-         'default': None},
-        {'name': 'dichroic_model_number',
-         'type': str,
-         'doc': 'dichroic model number',
-         'default': None},
-        allow_extra=True)
+@docval(
+    {
+        'name': 'photodetector',
+        'type': int,
+        'doc': 'references rows of PhotodetectorsTable',
+        'default': None
+    },
+    {
+        'name': 'location',
+        'type': str,
+        'doc': 'location of fiber',
+        'default': None
+    },
+    {
+        'name': 'notes',
+        'type': str,
+        'doc': 'description of fiber',
+        'default': None
+    },
+    {
+        'name': 'fiber_model_number',
+        'type': str,
+        'doc': 'fiber model number',
+        'default': None
+    },
+    {
+        'name': 'dichroic_model_number',
+        'type': str,
+        'doc': 'dichroic model number',
+        'default': None
+    },
+    allow_extra=True
+)
 def add_fiber(self, **kwargs):
```

Comment:
I still think we should ditch/deprecate the `add_fiber` method in favor of `add_row` as suggested by @rly in #8.

Reply:
done!